src/components/CreateProfile/CreateProfileInput/CreateProfileInput.jsx

```diff
@@ -0,0 +1,11 @@
+'use client'
+import React from 'react'
+import styles from './CreateProfileInput.module.scss'
+
+const CreateProfileInput = ({ placeholder }) => {
```

Comment:
**nitpick:** This component should be more generic, like `TextInput` -  we are also missing required props here, like `onChange`, `value`. Exactly how you would do it in Nerdux